src/App.jsx

```diff
@@ -1,35 +1,107 @@
+    movie: "Brave, 2012",
+  },
+  {
+    text: "Always let your conscience be your guide.",
+    speaker: "The Blue Fairy",
+    movie: "Pinocchio, 1940",
+  },
+  {
+    text: "I never look back, darling! It distracts from the now.",
+    speaker: "Edna Mode",
+    movie: "The Incredibles, 2004",
+  },
+  {
+    text: "To infinity...and beyond!",
+    speaker: "Buzz Lightyear",
+    movie: "Toy Story, 1995",
+  },
+  {
+    text: "Oh yes, the past can hurt. But the way I see it, you can either run from it or learn from it.",
+    speaker: "Rafiki",
+    movie: "The Lion King, 1994",
+  },
+  {
+    text: "Even miracles take a little time.",
+    speaker: "Fairy Godmother",
+    movie: "Cinderella, 1950",
+  },
+  {
+    text: "The only thing predictable about life is its unpredictability.",
+    speaker: "Remy",
+    movie: "Ratatouille, 2007",
+  },
+  {
+    text: "A true hero isn't measured by the size of his strength but by the strength of his heart.",
+    speaker: "Hera",
+    movie: "Hercules, 1997",
+  },
+  {
+    text: "You are braver than you believe, steonger than you seem and smarter than you think.",
+    speaker: "Winnie The Pooh",
+    movie: "The Many Adventures of Winnie The Pooh, 1977",
+  },
+  {
+    text: "Sometimes our stengths lie beneath the surface.",
+    speaker: "Moana",
+    movie: "Moana, 2016",
+  },
+  {
+    text: "The only limit is your imagination.",
+    speaker: "Hiro",
+    movie: "Big Hero 6, 2014",
+  },
+  {
+    text: "The heart is not so easily changed but the head can be persuaded.",
+    speaker: "Grand Pabbie",
+    movie: "Frozen, 2013",
+  },
+];
 
 function App() {
-  const [count, setCount] = useState(0)
+  const getRandomQuote = () => {
+    const randomIndex = Math.floor(Math.random() * quotes.length);
+    return quotes[randomIndex];
+  };
+
+  const [currentQuote, setCurrentQuote] = useState(getRandomQuote());
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
+    <div className="container">
+      <div className="title">
+        <h2>Random Inspirational Disney Quotes</h2>
+      </div>
+      <div className="text">
+        <QuoteText quote={currentQuote} />
       </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
+      <div className="button">
+        <QuoteButton onClick={() => setCurrentQuote(getRandomQuote())} />
       </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    </div>
+  );
 }
 
-export default App
+export default App;
```

Comment:
![image](https://github.com/bpb2008/MagicQuotes/assets/152167/51f07ebc-53f4-47bb-bd88-42797eca9514)

Would you be up for doing these extra challenges? If yes, we can merge this PR's branch first and then you can create a new feature branch to work on these next parts.

Reply:
Sure!